src/components/idleMembers/card/index.tsx

```diff
@@ -28,7 +39,7 @@ const Card: FC<Props> = ({ idleMemberUserName }) => {
       aria-hidden="true"
     >
       <Image
-        src={isImageAvailable ? `/${DUMMY_PROFILE}` : assigneeProfilePic(idleMemberUserName)}
+        src={isImageAvailable ? `/${DUMMY_PROFILE}` : assigneeProfilePic}
```

Comment:
Is this sequence looking right? I thought if we have an image available, we want to be showing the `assigneeProfilePic`?

Reply:
yeah, the sequence was incorrect. Resolved in the latest PR. Thanks